migration-tool/src/main/java/software/amazon/awssdk/migration/internal/recipe/V1SetterToV2.java

```diff
@@ -37,10 +37,13 @@
  * for generated model classes and client classes.
  *
  * @see NewClassToBuilderPattern
+ * TODO: separate model classes and client classes
  */
 @SdkInternalApi
 public class V1SetterToV2 extends Recipe {
     private static final Map<String, String> CLIENT_CONFIG_NAMING_MAPPING =
+        // TODO: handle other settings on the builder such as withEndpointConfiguration,
+        //  withMonitoringListener and withMetricsCollector
```

Comment:
Are the other settings included in the scope for the preview version, or will they come later? Is there an approximate estimate for the size?

Reply:
They will not be included in the preview version. It's not trivial to implement them, probably x-large since we don't have anything similar implemented at the moment. It may become easier as we go. We can prioritize them based on feedback when we get to it.